bin/pycbc_make_skymap

```diff
@@ -27,6 +27,8 @@ from ligo.gracedb.rest import GraceDb
 def default_frame_type(time, ifo):
     """Sensible defaults for frame types based on interferometer and time.
     """
+    if opt.fake_strain:
+        return ifo + '_FAKE_DATA'
```

Comment:
I forget if we discussed this already; could you explain why this is needed? If one wants fake strain, specifying the frame type should not be necessary (i.e. this function should not even be called). Same comment for `default_channel_name()` below.

Reply:
The frame_type indeed is not needed. The channel name is needed (weirdly) because it is what is used to figure out the ifo. I don't like that that is the case, but I think this is not the place to change that.